src/Network/Statsd.hs

```diff
@@ -17,141 +14,42 @@ module Network.Statsd (
   histogram,
 ) where
 
-import Control.Monad
-import Control.Exception
+import Network.Statsd.UdpClient(UdpClient(..), fromURI, send)
```

Comment:
I think we can remove the field import `(..)` from UdpClient as it doesn’t export anything beyond the type identifier.